server/src/main/java/org/opensearch/index/shard/RemoteStoreRefreshListener.java

```diff
@@ -254,6 +255,11 @@ private boolean syncSegments() {
                     Map<String, Long> localSegmentsSizeMap = updateLocalSizeMapAndTracker(localSegmentsPostRefresh).entrySet()
                         .stream()
                         .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+
+                    Collection<String> segmentsToRefresh = localSegmentsPostRefresh.stream()
```

Comment:
Are we using this collection?

Reply:
No. Missed removing it. Will do so in the followup revision